doc/classes/ProjectSettings.xml

```diff
@@ -766,15 +766,13 @@
 		<member name="display/window/ios/allow_high_refresh_rate" type="bool" setter="" getter="" default="true">
 			If [code]true[/code], iOS devices that support high refresh rate/"ProMotion" will be allowed to render at up to 120 frames per second.
 		</member>
-		<member name="display/window/ios/hide_home_indicator" type="bool" setter="" getter="" default="true">
-			If [code]true[/code], the home indicator is hidden automatically. This only affects iOS devices without a physical home button.
-		</member>
 		<member name="display/window/ios/hide_status_bar" type="bool" setter="" getter="" default="true">
-			If [code]true[/code], the status bar is hidden while the app is running.
+			If [code]true[/code], the status bar is hidden while the app is running. This can be changed at runtime with `DisplayServer.ios_set_status_bar_appearance`.
```

Comment:
```suggestion
			If [code]true[/code], the status bar is hidden while the app is running. This can be changed at runtime with [method DisplayServer.ios_set_status_bar_appearance].
```